### PR TITLE
Updated Data updating based on #26 #11

### PR DESCRIPTION
**Pull request by jumper385**
created a code example for it - this is there to prevent confusion and to offer one more more code example!
Please let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,28 @@ Some restrictions to the provided data:
 * You can only add new data. Once you call `update`, you can not edit or delete existing data.
 * The x value of each data point must be monotonically increasing.
 * Due to the limitation of single-precision floating-point numbers, if the absolute value of x is large (e.g. `Date.now()`), you may need to use `baseTime` option  (see below) to make the chart render properly.
+```JavaScript
+let starttime = (new Date()).getTime(); // Set the start time e.g. 1626186924936
```
Comment (huww98) [inline README.md:136]:
```suggestion
let starttime = Date.now(); // Set the start time e.g. 1626186924936
```
This should give the same result with fewer code

**Review by huww98 — CHANGES_REQUESTED** [review]

**Review by jumper385 — COMMENTED** [review]
happy with this change

**Comment by jumper385** [comment]
I've accepted your change request and committed the changes to `patch-1` of my repo but it still seems like the change hasn't gone through... do you see the change on your side?

**Comment by huww98** [comment]
Yes, It seems OK. Thanks.

**Comment by jumper385** [comment]
PERFECT! Glad to help! I’ll try to help more with the README and some code too!

**Pull request merged**